python/ops.py

```diff
@@ -295,37 +434,48 @@ def splat_non_uniform(x, bins):
   method takes NumPy arrays instead of TF tensors, and returns a Numpy array.
 
   Args:
-    x: A 1D vector of values being splatted, in the shape of [batch_size].
+    x: A 2D matrix of values being splatted, in the shape of [batch_size, 1].
     bins: 1D vector, in the shape of [N], where N is the number of bins.
 
   Returns:
-    A 2D matrix with interpolated weights in the shape of [batch_size, M], where
-    each row is the splat weights.
+    f: A 2D matrix with interpolated weights in the shape of [batch_size, N],
+      where each row is the splat weights.
   """
 
-  x = np.asarray(x)
-  bins = np.asarray(bins)
+  N = tf.size(bins)
```

Comment:
Not a good variable name, let's change that.

Reply:
Changed to bin_num